app/build.gradle

```diff
@@ -53,10 +53,12 @@ android {
         standard {
             dimension 'version'
             applicationId "com.KillerBLS.modpeide"
+            resValue "string", "app_name", "ModPE IDE"
         }
         ultimate {
             dimension 'version'
             applicationId "com.LightTeam.modpeidepro"
+            resValue "string", "app_name", "ModPE IDE Pro"
```

Comment:
Okay I'll merge it, but later I change these lines back because this sounds a little confusing. There is no "Pro" version anymore, only Standard and Ultimate.

I agree that it looks weird because you can't tell for sure where is free and where is paid version, but that's another issue